### PR TITLE
Be able to preview events

### PR DESCRIPTION
**Pull request by jennysivapalan**
This is to help the Events team preview draft events before launching.

As we don't want to expose this to everyone there is a new endpoint which is `/preview-event/{id}`. This endpoint:
- requires Google auth (so must have a guardian email). 
- directly calls the Eventbrite API to get the most recent details of the event
- has a short cache of 1 second (to get updates quickly).

I've only applied this to Guardian Live events and not Masterclasses. (We can add this later if they need but it requires more work for figuring which service to call.)

I've pushed some of the logic on which status text to display from the template to the Scala code. This makes it testable. The downside is there is a horrible if condition. Sorry @wpf500 

@jamesoram let me know when you're free and we can go through the testing of this. 

Did I mention I'm going on holiday? :ski:

### CODE REVIEW

File: frontend/app/model/Eventbrite.scala [hunk 1]
```diff
@@ -102,7 +102,13 @@ object Eventbrite {
...
     val isNoTicketEvent = description.exists(_.html.contains("<!-- noTicketEvent -->"))
     val isBookable = status == "live" && !isSoldOut
-    val isPastEvent = status != "live"
+    val isPastEvent = status != "live" && status != "draft"
+
+    val statusText =
+      if(isPastEvent) "Past event"
+      else if(isSoldOut) "Sold out"
+      else if(status == "draft") "Preview of Draft Event"
+      else ""
```
Comment (wpf500) [inline frontend/app/model/Eventbrite.scala]: Not a massive fan but seen as you're going on holiday! :ski:

**Comment by wpf500** [comment]
:+1:

File: frontend/app/model/Eventbrite.scala (hunk 1 above)
Comment (jennysivapalan) [inline frontend/app/model/Eventbrite.scala]: No, it's rubbish. but not sure how to improve this :put_litter_in_its_place:

**Comment by jamesoram** [comment]
:+1:

**Comment by rtyley** [comment]
Looks great!

https://membership.theguardian.com/preview-event/15351670259
https://membership.theguardian.com/preview-event/15355009246

:+1: :+1: :+1: :+1: :+1:

**Pull request merged**